heron/instance/src/java/com/twitter/heron/instance/AbstractOutputCollector.java

```diff
@@ -37,6 +37,7 @@
   private long totalTuplesEmitted;
   private PhysicalPlanHelper helper;
 
+  @SuppressWarnings("deprecation")
```

Comment:
Where we add this, we should add a comment that it's only need until TOPOLOGY_ENABLE_ACKING is removed. That way it doesn't linger after TOPOLOGY_ENABLE_ACKING is removed. Same for other occurrences in this pr.